src/components/button-toggle/button-toggle.pw.tsx

```diff
@@ -425,8 +408,9 @@ test.describe("Accessibility tests", () => {
   });
 
   ([
-    [SIZE.SMALL, 32],
-    [SIZE.LARGE, 48],
+    [SIZE.SMALL, 24],
+    [SIZE.MEDIUM, 32],
```

Comment:
**question:** is it right that these `MEDIUM` tests have snuck back in since the QA review?

Reply:
Yeah, button size of Medium is valid, it was buttonIconSize of medium that needed to be removed :)